packages/client/src/components/reports/scape/ScapeExecutionsInProgressResult.tsx

```diff
@@ -0,0 +1,250 @@
+
+    return (
+      <div className="scape-in-progress-report-container">
+        <div className="scape-metadata-container">
+          <TitledSection title="Canary Report" />
+          <ScapeMetadataSection
+            application={application as string}
+            user={user as string}
+            applicationMetadata={applicationMetadata as KvMap<string>}
+            metricsAccountName={metricsAccountName as string}
+            storageAccountName={storageAccountName as string}
+            startTime={startTime as string}
+            endTime={stageStatus.IN_PROGRESS}
+            lifetime={lifetime as number}
+            request={request as CanaryAnalysisExecutionRequest}
+            scope={safeGet(() => request.scopes[0]).get() as CanaryAnalysisExecutionRequestScope}
+            canaryConfig={canaryConfig as CanaryConfig}
+            handleGoToConfigButtonClick={handleGoToConfigButtonClick}
+          />
+        </div>
+        <div className="message-bar">Canary Analysis In Progress</div>
+        <div className="scape-executions">
+          <div className="scape-executions-tabs">
+            {runCanaryStages.map((stageMetadata, index) => {
+              const refereeStageMetadata: RefereeStageMetadata = determineRefereeStageMetadata(stageMetadata, index);
+
+              return (
+                <div className="scape-tab-wrapper" key={stageMetadata.name}>
+                  <div
+                    key={`execution-${index}`}
+                    className={[
+                      'btn',
+                      'scape-tab',
+                      refereeStageMetadata.status,
+                      index === this.state.selectedCAEIndex ? 'selected' : 'not-selected'
+                    ].join(' ')}
+                    onClick={() => {
+                      this.onCAETabClick(index, refereeStageMetadata.status);
+                      if (refereeStageMetadata.availableResult) {
+                        handleCanaryRunSelection(refereeStageMetadata.availableResult);
+                      }
+                    }}
+                  >
+                    <div className="score-wrapper headline-md-marketing">
+                      <div className="score">{refereeStageMetadata.score}</div>
+                      <div className="label uppercase">
+                        {refereeStageMetadata.status === stageStatus.IN_PROGRESS ? '' : refereeStageMetadata.status}
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              );
+            })}
+          </div>
+          <div className="scape-executions-result-wrapper">
+            {this.state.isSelectedRunComplete ? (
+              isTerminalFailure(selectedCanaryExecutionResult) ? (
+                <div className="terminal-canary-wrapper">
+                  <TerminalResult exception={selectedCanaryExecutionResult.exception} />
+                </div>
+              ) : (
+                <ScapeRunResult
```

Comment:
Consider using https://reactjs.org/docs/jsx-in-depth.html#spread-attributes to pass props to this component.